packages/react-composites/tests/browser/chat/hermetic/FileSharing.test.ts

```diff
@@ -35,12 +35,16 @@ test.describe('Filesharing SendBox', async () => {
           {
             name: 'SampleFile.pdf',
             extension: 'pdf',
-            url: 'https://sample.com/SampleFile.pdf'
+            url: 'https://sample.com/SampleFile.pdf',
+            attachmentType: 'fileSharing',
```

Comment:
don't think existing test should be modified, these are for acs chat and doesn't apply for interop (noticing how this test is about uploading)

Reply:
I have been getting a lot of feedback from different folks about this. The general consensus is instead of introducing a new `ChatAttachment` type like I was doing before which is  just a superset of an existing `Filemetadata` objet, we merge the two together. So thats what I have done. We are most likely going to rename the concept of `FileUploadAdapter` cause the name won't be as applicable. But that can happen in a future PR.